var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -111,6 +111,8 @@ class Paraview(CMakePackage, CudaPackage):
     conflicts("+python3", when="@:5.5")
     conflicts("+shared", when="+cuda")
     conflicts("+cuda", when="@5.8:5.10")
+    conflicts("+rocm", when="+cuda")
```

Comment:
why not hip as opposed to rocm. Note that CUDA is to CUDA TK what HIp is to ROCM.

Reply:
Nevermind, I can see that this variant is provided by the ROCmPackage